apps/client/src/common/api/ontimeApi.ts

```diff
@@ -112,29 +113,51 @@ export async function postOscSubscriptions(data: OscSubscription) {
  * @description HTTP request to download db
  * @return {Promise}
  */
-export const downloadRundown = async () => {
-  await axios({
+export const downloadRundown = async (exportType: 'csv' | 'json') => {
+  const response = await axios({
```

Comment:
great work.

I think that the strategies should be separate, the CSV file is technically not related to the project, but a feature.
With that in mind, the code related to it should be detachable from the remaining code

Could we have separate `downloadRundown` and `downloadCSV` functions, potentially with a auxiliary function for shared logic (if any)?